.env.example

```
@@ -92,6 +92,9 @@ GCP_STATIC_BUCKET_NAME=
 GCP_PROJECT_ID=
 GCP_BUCKET_LOCATION=
 
+# Expiration time of signed urls used to upload files to Google Cloud Platform
+SIGNED_URL_LIFETIME_IN_MINUTES=
```

I'd add a tiny bit more context to this variable:
```suggestion
GCS_SIGNED_URL_LIFETIME_IN_MINUTES=
```